db.go

```diff
@@ -14,6 +14,8 @@ const (
 	//   - pure go
 	//   - stable
 	GoLevelDBBackend BackendType = "goleveldb"
+	// same to GoLevelDBBackend but readonly
+	GoLevelDBBackendReadOnly BackendType = "goleveldb_readonly"
```

Comment:
Can you also explain why you need this change? Thank you!

Reply:
thank u for reply.
tm-db creates the dbCreator list when the module is init, this can easily select the readonly dbCreator

Reply:
maybe i should make a issue